rest_framework_simplejwt/settings.py

```diff
@@ -34,12 +34,16 @@
     "SLIDING_TOKEN_REFRESH_EXP_CLAIM": "refresh_exp",
     "SLIDING_TOKEN_LIFETIME": timedelta(minutes=5),
     "SLIDING_TOKEN_REFRESH_LIFETIME": timedelta(days=1),
+    "SLIDING_TOKEN_OBTAIN_SERIALIZER": "rest_framework_simplejwt.serializers.TokenObtainSlidingSerializer",
+    "TOKEN_OBTAIN_SERIALIZER": "rest_framework_simplejwt.serializers.TokenObtainPairSerializer",
 }
 
 IMPORT_STRINGS = (
     "AUTH_TOKEN_CLASSES",
     "TOKEN_USER_CLASS",
     "USER_AUTHENTICATION_RULE",
+    "SLIDING_TOKEN_OBTAIN_SERIALIZER",
```

Comment:
It could just be awhile, but aren't there more views with other serializers? I think this functionality would just need to add support for all views.

Reply:
LGTY I'll commit to adding support for other serializers as well.